src/reducers/VaccineReducer.js

```diff
@@ -1,14 +1,78 @@
 import { VACCINE_ACTIONS } from '../actions/VaccineActions';
 
+const LAST_DOWNLOAD_STATUS = {
+  OK: 'OK',
+  ERROR: 'ERROR',
+};
```

Comment:
OK - I am keen for ideas of what this really should be.. 🤷 

At the moment just want to show the `WiFiIcon` or `NoWifiIcon` so two states is fine

- could make it `LAST_DOWNLOAD_OK` with a boolean.... but reasonably certain we might want to put a reason at some point - "Couldn't find sensor", "Couldn't connect with sensor". 
- So maybe it should be LAST_DOWNLOAD_ERROR and be `null` if it's OK? 
- Then I thought having "OK" was nicer than `null` to represent `OK` - ok?

Reply:
The current enum meets requirements, so I think all good to have as is.
We can extend when new scenarios come up - to your point, Kat, we could have `OK` and `everything else` - so we show the ok icon if the status is ok and an error state otherwise. 
What sort of error conditions could there be? ( I see little value in second guessing possible states.. all I've seen is 'sensor disconnected' / 'connection error' sort of things. )

Also, why is it a wifi icon? these are bluetooth sensors! ( this has already confused me )